packages/zowe-explorer/src/abstract/ZoweTreeProvider.ts

```diff
@@ -169,6 +169,7 @@ export class ZoweTreeProvider {
                 await setSession(node, EditSession as ISession);
             } else {
                 this.deleteSessionByLabel(node.label);
+                this.mHistory.addSession(node.label);
```

Comment:
Adding L172 ` this.mHistory.addSession(node.label);` seems to have fixed the issue where the profile was getting removed out of the `settings.json` file after editing, but I'm a little confused what lines 171-172 are meant to be doing now. It looks like L171 just removes the session from `settings.json` and then L172 just adds it back in and cancels L171 out. Could these two lines just be removed?